src/Components/ProductFootprint/details_old.jsx

```diff
@@ -0,0 +1,122 @@
+
+
+// importing react features to use states, effects, and links
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+// import database
+import { supabase } from '../../supabaseClient';
+
+// declare + export supplierManagement component
+export default function Details() {
+  // tableData - state var that stores all table data, initially an empty array
+  // setTableData - function used to update value of tableData
+  const [tableData, setTableData] = useState([]);
+  // isPopupOpen - state var that changes depending on whether popup is open or not, initially set to false
+  // setIsPopupOpen - function used to update the value of isPopupOpen
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
+  // newRowData - state var that stores obj containing properties for new row of data, all props initially set to empty string
+  // setNewRowData - function to set new row data
+  const [newRowData, setNewRowData] = useState({ lca_name: '', mass_units: '', dist_units: '', vol_units: ''});
+
+  // useEffect - effect that runs once when component mounts
+  useEffect(() => {
+    // call fetchMeasurement function
+    fetchMeasurement()
+  }, []) // <-- '[]' as the dependency array means it runs once
+
+  // fetches data from db + updates tableData var w/ fetched data
+  async function fetchMeasurement() {
```

Comment:
rename this function. And if possible move this database function call to "generalFunction.jsx".

Reply:
Not using this anymore. Will delete the file.